scripts/exwave_post_sbs.sh

```diff
@@ -582,25 +541,28 @@
       if [ "$DOGRI_WAV" = 'YES' ]
       then
         nigrd=1
+        if [ ${CFP_MP:-"NO"} = "YES" ]; then nm=0 ; fi # Counter for MP CFP
```

Comment:
This line is not necessary since you never use $nm in this if-block/loop. I guess you aren't using a command file for this section? Does this section work on Hera? If so, remove this line.

Reply:
I've removed the nm entries in deprecated blocks. Good eye!